### ISSUE TITLE
setuptools warning

### ISSUE THREAD

**Issue by asarubbo** [dalibo/pg_activity#411]
By packaging 3.5.0 on Gentoo Linux I see the following setuptools warning:

```
 * QA Notice: setuptools warnings detected:
 * 
 *      Package 'pgactivity.profiles' is absent from the `packages` configuration.
 *      Package 'pgactivity.queries' is absent from the `packages` configuration.
```

**Comment by dlax** [comment]
What command do you run to get these warnings?

**Comment by dlax** [comment]
Based on https://setuptools.pypa.io/en/latest/userguide/datafiles.html it's not clear to me what would be missing in our setup.

**Comment by asarubbo** [comment]
> What command do you run to get these warnings?

the warning comes from portage that search for 'setuptools warning' string:
https://github.com/gentoo/portage/blob/08a2bc3800ea2e997716903244ec041339b45b06/lib/portage/package/ebuild/doebuild.py#L2395

The exact command used to build is:
```
python3.12 -m gpep517 build-wheel --prefix=/usr --backend setuptools.build_meta --output-fd 3 --wheel-dir /var/tmp/portage/dev-db/pg_activity-3.5.0/work/pg_activity-3.5.0-python3_12/wheel
```

For completeness I'm attaching the full build log:
[pg_activity-3.5.0_20240403-070656.log.txt](https://github.com/dalibo/pg_activity/files/14848259/pg_activity-3.5.0_20240403-070656.log.txt)

An interesting part is:
```
/usr/lib/python3.12/site-packages/setuptools/command/build_py.py:207: _Warning: Package 'pgactivity.profiles' is absent from the `packages` configuration.
!!

        ********************************************************************************
        ############################
        # Package would be ignored #
        ############################
        Python recognizes 'pgactivity.profiles' as an importable package[^1],
        but it is absent from setuptools' `packages` configuration.

        This leads to an ambiguous overall configuration. If you want to distribute this
        package, please make sure that 'pgactivity.profiles' is explicitly added
        to the `packages` configuration field.

        Alternatively, you can also rely on setuptools' discovery methods
        (for example by using `find_namespace_packages(...)`/`find_namespace:`
        instead of `find_packages(...)`/`find:`).

        You can read more about "package discovery" on setuptools documentation page:

        - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html

        If you don't want 'pgactivity.profiles' to be distributed and are
        already explicitly excluding 'pgactivity.profiles' via
        `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
        you can try to use `exclude_package_data`, or `include-package-data=False` in
        combination with a more fine grained `package-data` configuration.

        You can read more about "package data files" on setuptools documentation page:

        - https://setuptools.pypa.io/en/latest/userguide/datafiles.html


        [^1]: For Python, any directory (with suitable naming) can be imported,
              even if it does not contain any `.py` files.
              On the other hand, currently there is no concept of package data
              directory, all directories are treated like packages.
        ********************************************************************************
```

**Comment by dlax** [comment]
I can reproduce indeed; thank you.
Will look at it.

**Comment by dlax** [comment]
Just released version 3.5.1 with this and #410 fixed.
